# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/firefox/hotfix-no-avx.patch.x86
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- firefox-106.0.3/mozglue/misc/moz.build.vanilla	2022-11-03 17:27:41.842048117 +0100
+++ firefox-106.0.3/mozglue/misc/moz.build	2022-11-03 17:30:48.162041567 +0100
@@ -51,7 +51,7 @@
     "Uptime.cpp",
 ]
 
-if CONFIG["CPU_ARCH"].startswith("x86"):
+if CONFIG["CPU_ARCH"].startswith("x86_"):
     SOURCES += [
         "SIMD_avx2.cpp",
         "SSE.cpp",
